styles.css

```
@@ -0,0 +1,58 @@
+.title-dr{
+    text-align: center;
+    font-size: 40px;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif;
+}
+.title-reason{
+    text-align: center;
+}
+
+#div-img{
+    text-align: center;
+    max-width: 100%;
+    display: flex;
+    background-color: white;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    justify-content: center;
+}
+#img-caption{
+    text-align: center;
+    padding-top: 50px;
+}
+.story{
+    text-align: center;
+}
+.list{
+    width: 500px;
+    height: fit-content;
+    line-height: 1.5;
+    align-content: center;
+    margin-left: 380px;
+    margin-top: 40px;
+}
+li{
```

```suggestion
.list li
```
xài css selector để ko ảnh hưởng chỗ khác